apps/dashboard/components/array-input.tsx

```
@@ -15,85 +12,74 @@ type Props = {
-              </Badge>
-            );
-          })}
-          {/* Avoid having the "Search" Icon */}
-          <CommandPrimitive.Input
-            ref={inputRef}
+    <div className="bg-transparent flex-col relative max-w-96">
+      <div className="flex flex-row gap-2">
+        <div className="flex flex-col justify-end">
+          {title && <span className="text-xs font-medium mb-2 ">{title}:</span>}
+        </div>
+        <div className="flex flex-wrap gap-1 mb-2" role="list" aria-label="Selected items">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<ol>
<ul>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)